docs/arcaflow/index.md

```diff
@@ -36,21 +36,23 @@ You need to pick the right tool for the job. Sometimes, you need something simpl
     - It can run things on remote systems out of the box
     - It runs tasks as sequential (and, with some work, parallel) steps
     - You can pass data between steps
+    - Plugins and modules can be written in a variety of languages
+    - Users can define their own connection plugins
+    - Tasks can run inside containers using custom connection plugins ([k8s example](https://docs.ansible.com/ansible/latest/collections/kubernetes/core/kubectl_connection.html))
 
     **What it does better than Arcaflow**
     
     - Well established and well known
     - Wide range of available plugins
     - Simple workflows are easy to write
-    - It runs comands over SSH by design
+    - It runs comands over SSH by default
     
     **What it does worse than Arcaflow**
     
-    - Parallelization is hard, it is intended for resource creation rather than generic workflows
-    - It does not have a typing system and cannot guarantee type-safety or validation of data
-    - Plugins can only be written in Python
-    - Plugins are not transparently runnable in a container engine as Ansible intends to run commands over SSH
-    - It is written in Python and needs a large number of dependencies installed
+    - Ansible's approach to parallelization is in terms of executing the same tasks against different hosts in parallel (see "forks" and "strategy"). Defining different tasks to perform in parallel is more challenging.
+    - By default, input/output data is not strongly typed. Any data type validation must be done explicitly within playbooks.
```

Comment:
I think one point that is less obvious with this wording is the type validation doesn't happen until the step is run, whereas with the type system in arcaflow, it can validate it without executing the steps.

This is a critical point. It's like using Python/Javascript vs Java/C++/Rust/TypeScript.